pages/privacy.md

```diff
@@ -0,0 +1,125 @@
+personally identifiable information, including your name, email address, and
+country of residence. If you are a supporting or sponsoring member of the SCF,
+or wish to make a donation, we will also ask for the information necessary to
+process a payment and reduce our risk of fraud. We may ask for other
+information, but providing such information is completely optional.
+
+Voting Members will need to provide us with email contact information during
+their enrollment. However, that information is never made generally available
+outside of the Foundation, neither on this website nor anywhere else public.
+
+## 2. What do we use this information for?
+
+**Basic usage information.** We use basic usage information to protect against
+**unauthorized intrusion into SCF-controlled servers, as well as monitor and
+**improve the availability and performance of SCF-controlled websites. We may
+**also use information collected from you to personalize your interactions with
+**SCF-controlled websites.
+
+**Personally identifiable information.** We use contact information to
+**communicate with members or with others that have accounts with the SCF, to
+**provide access to secure areas of our websites, and to provide the services
+**identified on our sites. We also use the information we collect to improve the
+**content of our websites We also use this information to prevent or detect
+**fraud or abuses of our websites and to enable third parties to carry out
+**technical, logistical or other functions on our behalf.   Financial
+**information and payment card details. We may ask those who want to donate to
+**the SCF, become a sponsor member, or become a supporting member to enter
+**credit card or account information in order to process payments to the SCF.
+**Financial information provided via our websites is transferred over a secure
+**connection. We collect this information on a transaction-by-transaction basis
+**and will not authorize the release of the information with anyone not directly
+**involved in processing the transaction. This information may be encrypted and
+**stored for logging purposes in accordance with applicable regulatory
+**requirements. Credit card numbers are used only for processing payments and
+**are not used for any other purposes. When we use third party service providers
+**for credit card processing services we will enter into a written agreement
+**with such third party ensuring appropriate security and confidentiality
+**measures are in place in safeguarding your personal data.
+
+## 3. Disclosure of personal information
+
+We will not disclose any information we collect to any third party unless it is:
+
+1. Specifically authorized by you; or 2. Necessary to perform the functions
+requested by you (and only to the extent absolutely necessary to perform the
+function); or 3. Required to check for fraud or other misuse of SCF resources;
+or 5. Required by law.
```

Comment:
1, 2, 3, 5 - maybe just break this out into an enumerated list and let Markdown do the numbering?